basicsynbio/functions/pdf_instructions.py

```diff
@@ -0,0 +1,409 @@
+        )
+    )
+    ## Magbead Purification 2
+    elems.append(Paragraph("Magbead purification", styles["Heading2"]))
+    elems.append(
+        Paragraph(
+            "Prepare fresh 70% EtOH (0.5 ml per BASIC reaction) and bring magnetic beads (AmpureXP or Ampliclean) stored at 4°C back into homogeneous mix by shaking thoroughly. ",
+            styles["BodyText"],
+        )
+    )
+    elems.append(
+        Paragraph(
+            "We recommend using a 96 well Falcon plate (Falcon 351177) in combination with an Ambion magnetic plate (AM10050) for quick magbead immobilisation and easy pipetting access. ",
+            styles["BodyText"],
+        )
+    )
+    elems.append(
+        ListFlowable(
+            [
+                Paragraph(
+                    "Add 54 μl of magnetic beads into 96 well Falcon plate (one well per BASIC reaction) and add the 30 μl BASIC linker ligation from the PCR machine step, mix by pipetting 10 times. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Wait 5min to allow DNA binding to magbeads. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Place Falcon plate on magnetic stand and wait for rings to form and solution to clear.",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Remove the solution with a 200 μl pipette tip from the centre of each well.",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Add 190 μl 70'%' EtOH to each well and wait 30 s. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Remove solution from each well (pipette set to 200 μl volume) ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Add 190 μl 70'%' EtOH to each well and wait 30 s. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Remove solution from each well (pipette set to 200 μl volume)",
+                    styles["BodyText"],
+                ),
+                Paragraph("Leave the plate to dry for 1-2 min.", styles["BodyText"]),
+                Paragraph(
+                    "Remove Falcon plate from magnet and resuspend magbeads in 32 μl dH20. ",
+                    styles["BodyText"],
+                ),
+                Paragraph("Wait 1 min for DNA to elute. ", styles["BodyText"]),
+                Paragraph(
+                    "Place Falcon plate back on magnetic stand and allow ring to form and solution to clear. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Pipette 30 μl of H20 with eluted DNA into fresh 1.5 ml Eppendorf tube for direct use in assembly or storage at -20°C for up to 1 month. ",
```

Comment:
"Transfer 30 μl of purified clip reaction into a clean microcentrifuge tube or well and store at at -20°C for up to 1 month."